include/boost/gil/extension/numeric/affine.hpp

```diff
@@ -108,6 +108,36 @@ boost::gil::matrix3x2<T> inverse(boost::gil::matrix3x2<T> m)
     return res;
 }
 
+//feature
+//rotate an image from its center point for -180 deg <= theta <= 180 deg
+//using consecutive affine transformations
+//As top-left corner of source image is taken as origin, idea is to rotate the source image about the origin and then translate the new image
+//according to the mentioned rotation angle, and to re-center and fit the image in the source dimensions 
+// preferably use type double for less precision errors
+template<typename T, typename F> boost::gil::matrix3x2<F> center_rotate(boost::gil::point<T> dims,F rads)
```

Comment:
```cpp
template<typename T, typename F>
boost::gil::matrix3x2 ...
```

Doxygen documentation of the function prototype is missing (e.g. what is `F`, etc.)